bundles/framework/publisher2/view/form/MapPreviewForm.jsx

```diff
@@ -31,7 +29,7 @@ const RowContainer = styled('div')`
 
 const Label = (props) => {
     const { option } = props;
-    let text = Oskari.getMsg(MESSAGE_BUNDLE_KEY, 'BasicView.sizes.' + option.id);
+    let text = Oskari.getMsg(PUBLISHER_BUNDLE_ID, 'BasicView.sizes.' + option.id);
     if (option.id !== CUSTOM_MAP_SIZE_ID && !isNaN(parseInt(option.width)) && !isNaN(parseInt(option.height))) {
         text += ' (' + option.width + ' x ' + option.height + 'px)';
     }
```

Comment:
Something like this might work so we don't have to keep passing the bundle id and use Oskari.getMsg() directly:
```suggestion
    let extraInfo = '';
    if (option.id !== CUSTOM_MAP_SIZE_ID && !isNaN(parseInt(option.width)) && !isNaN(parseInt(option.height))) {
        extraInfo = '(' + option.width + ' x ' + option.height + 'px)';
    }
    return <Message key={ 'BasicView.sizes.' + option.id}>{ extraInfo }</Message>
```